applications/GeoMechanicsApplication/custom_elements/small_strain_U_Pw_diff_order_element.cpp

```diff
@@ -785,14 +788,15 @@ void SmallStrainUPwDiffOrderElement::Calculate(const Variable<Vector>& rVariable
     }
 }
 
-Vector SmallStrainUPwDiffOrderElement::CalculateInternalForces(ElementVariables& Variables,
-                                                               const std::vector<Matrix>& b_matrices,
-                                                               const std::vector<double>& integration_coefficients,
-                                                               const std::vector<double>& biot_coefficients,
-                                                               const std::vector<double>& degrees_of_saturation,
-                                                               const std::vector<double>& biot_moduli_inverse,
-                                                               const std::vector<double>& relative_permeability_values,
-                                                               const std::vector<double>& bishop_coefficients)
+Vector SmallStrainUPwDiffOrderElement::CalculateInternalForces(
+    ElementVariables&          Variables,
+    const std::vector<Matrix>& b_matrices,
+    const std::vector<double>& integration_coefficients,
+    const std::vector<double>& biot_coefficients,
+    const std::vector<double>& degrees_of_saturation,
+    const std::vector<double>& biot_moduli_inverse,
+    const std::vector<double>& relative_permeability_values,
+    const std::vector<double>& bishop_coefficients) const
```

Comment:
Something that you don't need to do right away (since it's not that important), but these parameter names are not following the Kratos Style Guide:
```suggestion
    ElementVariables&          rVariables,
    const std::vector<Matrix>& rBMatrices,
    const std::vector<double>& rIntegrationCoefficients,
    const std::vector<double>& rBiotCoefficients,
    const std::vector<double>& rDegreesOfSaturation,
    const std::vector<double>& rBiotModuliInverses,
    const std::vector<double>& rRelativePermeabilityValues,
    const std::vector<double>& rBishopCoefficients) const
```
This suggestion also appliesto `SmallStrainUPwDiffOrderElement::CalculateExternalForces`.